docs/installation/_index.md

```diff
@@ -11,6 +11,7 @@ aliases:
     - /getting-started/linux-install/
     - /installation/linux-install/
     - /installation/install/
+    - /getting-started/installation/
```

Comment:
did we get feedback on this?

Reply:
Not sure what you mean? I happened upon it when I clicked a link in an old intern tutorial google doc, and realized we don't have this one in the aliases but it might be in other external places besides this one google doc because it was the link for a while.